language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs

```diff
@@ -386,83 +399,102 @@ callProcessSilent cmd args = do
+        , "  \"compilerOptions\": {"
+        , "    \"target\": \"es5\","
+        , "    \"lib\": ["
+        , "      \"es2015\""
+        , "     ],"
+        , "    \"strict\": true,"
+        , "    \"noUnusedLocals\": true,"
+        , "    \"noUnusedParameters\": false,"
+        , "    \"noImplicitReturns\": true,"
+        , "    \"noFallthroughCasesInSwitch\": true,"
+        , "    \"outDir\": \"lib\","
+        , "    \"module\": \"commonjs\","
+        , "    \"declaration\": true,"
+        , "    \"sourceMap\": true"
         , "    },"
-        , "    \"plugins\": ["
-        , "      \"@typescript-eslint\""
-        , "    ],"
-        , "    \"rules\": {"
-        , "      \"@typescript-eslint/no-floating-promises\": \"error\","
-        , "      \"@typescript-eslint/explicit-function-return-type\": \"off\","
-        , "      \"@typescript-eslint/no-inferrable-types\": \"off\""
-        , "    }"
-        , "  },"
-        , "  \"devDependencies\": {"
-        , "    \"@typescript-eslint/eslint-plugin\": \"^2.11.0\","
-        , "    \"@typescript-eslint/parser\": \"^2.11.0\","
-        , "    \"eslint\": \"^6.7.2\","
-        , "    \"typescript\": \"~3.7.3\""
-        , "  }"
+        , "  \"include\": [\"src/**/*.ts\"],"
         , "}"
         ]
 
-writeRootPackageJson :: FilePath -> String -> IO ()
-writeRootPackageJson rootDir workspace =
-   writeFileUTF8 (rootDir </> "package.json") $ unlines
-     [ "{"
-     , "  \"private\": true,"
-     , "  \"workspaces\": ["
-     , "    \"" <> workspace <> "\""
-     , "  ]"
-     , "}"
-     ]
+    writeEsLintConfig :: FilePath -> IO ()
+    writeEsLintConfig dir = writeFileUTF8 (dir </> ".eslintrc.json") $ unlines
+      [ "{"
+      , "  \"parser\": \"@typescript-eslint/parser\","
+      , "  \"parserOptions\": {"
+      , "    \"project\": \"./tsconfig.json\""
+      , "  },"
+      , "  \"plugins\": ["
+      , "    \"@typescript-eslint\""
+      , "  ],"
+      , "  \"extends\": ["
+      , "    \"eslint:recommended\","
+      , "    \"plugin:@typescript-eslint/eslint-recommended\","
+      , "    \"plugin:@typescript-eslint/recommended\","
+      , "    \"plugin:@typescript-eslint/recommended-requiring-type-checking\""
+      , "  ],"
+      , "  \"rules\": {"
+      , "    \"@typescript-eslint/explicit-function-return-type\": \"off\","
+      , "    \"@typescript-eslint/no-inferrable-types\": \"off\""
+      , "  }"
+      , "}"
+      ]
+
+    writePackageJson :: FilePath -> IO ()
+    writePackageJson dir = let name = takeFileName dir in writeFileUTF8 (dir </> "package.json") $ unlines
+            ["{"
+            , "  \"private\": true,"
+            , "  \"name\": \"@daml2ts/" <> name <> "\","
+            , "  \"version\": \"" <> sdkVersion <> "\","
+            , "  \"description\": \"Produced by daml2ts\","
+            , "  \"license\": \"Apache-2.0\","
+            , "  \"dependencies\": {"
+            , "    \"@daml/types\": \"" <> sdkVersion <> "\","
```

Comment:
Ultimately, we need to depend on `0.0.0-SNAPSHOT` here and include the library in a way similar to the `build-and-lint` test.

Reply:
Not sure about this. Let's discuss.

Reply:
If that discussion hasn't happened yet, I'd like to participate.

Reply:
I don't there's a lot to discuss. Here's my proposal on how to fix the issue: https://github.com/digital-asset/daml/pull/4688